scripts/backend_tests.py

```diff
@@ -202,7 +202,7 @@ def _convert_to_test_target(path):
                 os.path.join(base_path, root)))
         for subroot, _, files in os.walk(os.path.join(base_path, root)):
             for f in files:
-                if (f.endswith('_test.py') and
+                if (f.endswith('_test.py') or f.endswith('Test.py') and
```

Comment:
Actually `CodeClassifier.py` is classifier file name so test file name becomes `CodeClassifierTest.py`. That's why.